chart/templates/statsd/statsd-deployment.yaml

```
@@ -95,11 +95,22 @@ spec:
           {{- if $containerLifecycleHooks }}
           lifecycle: {{- tpl (toYaml $containerLifecycleHooks) . | nindent 12 }}
           {{- end }}
-          {{- if .Values.statsd.args }}
-          args: {{ tpl (toYaml .Values.statsd.args) . | nindent 12 }}
-          {{- else}}
           args:
-            - "--statsd.mapping-config=/etc/statsd-exporter/mappings.yml"
+          {{- if .Values.statsd.cache.size }}
+          - "--statsd.cache-size={{ .Values.statsd.cache.size }}"
```

Indent is incorrect here, lists should be indented by two spaces on top of the dash. Please apply also to the ones below.
```suggestion
            - "--statsd.cache-size={{ .Values.statsd.cache.size }}"
```